hack/generate-proto.sh

```diff
@@ -16,16 +16,18 @@
 
 set -e
 
-docker build -t gen-proto -f hack/proto/Dockerfile --target generate-files proto
+docker build --platform linux/amd64 -t gen-proto -f hack/proto/Dockerfile --target generate-files proto
```

Comment:
Do we really need to mention the platform switch?

Reply:
Oops.  I was trying to build from an M1 and it was failing.  I'll back this out.